src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderTestData.java

```diff
@@ -168,21 +170,55 @@ private static SnapshotBuilderProgramDataListOption generateSnapshotBuilderProgr
                       Person.RACE_CONCEPT_ID,
                       "Race",
                       List.of(new SnapshotBuilderProgramDataListItem().id(43).name("unused 3")))))
-          .featureValueGroups(
-              List.of(
-                  new SnapshotBuilderFeatureValueGroup().name("Condition"),
-                  new SnapshotBuilderFeatureValueGroup().name("Observation"),
-                  new SnapshotBuilderFeatureValueGroup().name("Procedure"),
-                  new SnapshotBuilderFeatureValueGroup().name("Surveys"),
-                  new SnapshotBuilderFeatureValueGroup().name("Person")))
           .datasetConceptSets(
               List.of(
+                  new SnapshotBuilderDatasetConceptSet()
+                      .name("Drug")
+                      .table(
+                          new SnapshotBuilderTable()
+                              .datasetTableName(DrugExposure.TABLE_NAME)
+                              .primaryTableRelationship("fpk_person_drug")
+                              .secondaryTableRelationships(
+                                  List.of(
+                                      "fpk_drug_concept",
+                                      "fpk_drug_type_concept",
+                                      "fpk_drug_route_concept",
+                                      "fpk_drug_concept_s"))),
+                  new SnapshotBuilderDatasetConceptSet()
+                      .name("Condition")
+                      .table(
+                          new SnapshotBuilderTable()
+                              .datasetTableName(ConditionOccurrence.TABLE_NAME)
+                              .primaryTableRelationship("fpk_person_condition")
+                              .secondaryTableRelationships(
+                                  List.of(
+                                      "fpk_condition_concept",
+                                      "fpk_condition_type_concept",
+                                      "fpk_condition_status_concept",
+                                      "fpk_condition_concept_s"))),
+                  new SnapshotBuilderDatasetConceptSet()
+                      .name("Procedure")
+                      .table(
+                          new SnapshotBuilderTable()
+                              .datasetTableName(ProcedureOccurrence.TABLE_NAME)),
+                  new SnapshotBuilderDatasetConceptSet()
+                      .name("Observation")
+                      .table(new SnapshotBuilderTable().datasetTableName(Observation.TABLE_NAME)),
+                  new SnapshotBuilderDatasetConceptSet()
+                      .name("Measurement")
+                      .table(new SnapshotBuilderTable().datasetTableName(Measurement.TABLE_NAME)),
```

Comment:
Minor comment: I could make a constant like I did for Measurement and use the table name like shown here, or just use the string table names like shown below (because they are only used once), or I could just define constants in this class